pkg/aggregator/no_aggregation_stream_worker.go

```diff
@@ -135,13 +148,24 @@ func (w *noAggregationStreamWorker) run() {
 						n := time.Now()
 						if serializedSamples > 0 && lastStream.Before(n.Add(-time.Second*1)) {
 							log.Debug("noAggregationStreamWorker: triggering an automatic payloads flush to the forwarder (no traffic since 1s)")
+							tlmNoAggFlush.Add(1)
 							break mainloop // end `Serialize` call and trigger a flush to the forwarder
 						}
 
 					// receiving samples
 					case samples := <-w.samplesChan:
 						log.Debugf("Streaming %d metrics from the no-aggregation pipeline", len(samples))
+						var processed int
+
 						for _, sample := range samples {
+							mtype, supported := metricSampleAPIType(sample)
+
+							if !supported {
+								log.Warnf("Discarding unsupported metric sample in the no-aggregation pipeline for sample '%s', sample type '%s'", sample.Name, sample.Mtype.String())
+								tlmNoAggSamplesProcessed.Add(1, noAggProcessStateUnsupportedType)
```

Comment:
Would it make sense to delay updating this telemetry value until after the loop, the same way is done for `ok` case?